agave_app/loadDialog.cpp

```diff
@@ -32,7 +32,7 @@ LoadDialog::LoadDialog(std::string path, const std::vector<MultiscaleDims>& dims
   std::string title;
   QString pathString = QString::fromStdString(path);
   QString extension = pathString.sliced(pathString.lastIndexOf("."));
-  if (pathString.indexOf("http") == 0) {
+  if ((pathString.indexOf("http") == 0) || (pathString.indexOf("s3:") == 0) || (pathString.indexOf("gs:") == 0)) {
```

Comment:
Is there a possibility there will be more of these in the future? Just thinking this looks like a place for a "isURL" function with approved prefixes

Reply:
I'm not aware of any yet but I agree this is about at the limit where if any new one comes up it will get refactored.